api/graphql/tests/test_recurring_reservations/test_recurring_reservations_queries.py

```diff
@@ -190,22 +190,21 @@ def test_filter_by_reservation_unit_name(self):
         ]
         for filter_name, filter_value, field_name in test_cases:
             response = self.query(
-                """
-                query {
-                    recurringReservations(%s: "%s") {
+                f"""
+                query {{
+                    recurringReservations({filter_name}: "{filter_value}") {{
```

Comment:
Having double curly braces like this is a bit ugly, so I prefer the format strings in this case. Although a better solution would be to use variables like [here](https://github.com/City-of-Helsinki/tilavarauspalvelu-core/blob/a6a7d4947520dc47d4074cf9d5faa0683bf48ffe/tests/test_graphql_api/test_banner_notification/test_filtering.py#L235). But we can leave that for later.